content/chronograf/v1.8/administration/managing-security.md

```diff
@@ -368,6 +369,27 @@ export HEROKU_ORGS=hill-valley-preservation-sociey,the-pinheads
 
     Note: If you’ve configured TLS/SSL, modify the `PUBLIC_URL` to ensure you're using HTTPS.
 
+#### Configure Bitbucket authentication
+
+1. Complete the instructions to [Use OAuth on Bitbucket Cloud](https://support.atlassian.com/bitbucket-cloud/docs/use-oauth-on-bitbucket-cloud/), including the following information:
+
+  - **Callback URL**: <http://localhost:8888/oauth/bitbucket/callback>
+  - **Permissions**: Account read, email
```

Comment:
@russorat, @sranka from reading Bitbucket's instructions, I'm not clear on how to set these "permissions" in line 377.
Bitbucket refers to `account` and `email` as "scopes"...do you include scopes in "grant types" and when "making requests"?

Reply:
my guess is that it's configured when you set up OAuth2 in the bitbucket UI. I think people who are familar with bitbucket will know where to set it.